frontends/election-manager/src/components/full_test_deck_tally_report_button.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useContext } from 'react';
+import { assert, tallyVotesByContest } from '@votingworks/utils';
+import { LogEventId } from '@votingworks/logging';
+import { Tally, VotingMethod } from '@votingworks/types';
+
+import { AppContext } from '../contexts/app_context';
+import { PrintButton } from './print_button';
+import { TestDeckTallyReport } from './test_deck_tally_report';
+import { generateTestDeckBallots } from '../utils/election';
+
+export function FullTestDeckTallyReportButton(): JSX.Element {
+  const { currentUserSession, electionDefinition, logger } =
+    useContext(AppContext);
+
+  assert(currentUserSession);
+  const currentUserType = currentUserSession.type;
+
+  assert(electionDefinition);
+  const { election } = electionDefinition;
+
+  const ballots = generateTestDeckBallots({ election });
+  const votes = ballots.map((b) => b.votes);
+  const testDeckTally: Tally = {
+    numberOfBallotsCounted: ballots.length * 4,
+    castVoteRecords: new Set(),
+    contestTallies: tallyVotesByContest({
+      election,
+      votes: [...votes, ...votes, ...votes, ...votes],
```

Comment:
> votes, Votes, VOTES, VOTES!!

How my mind reads this line 😆

Mind adding a comment here noting why we count 4 sets of votes, as you documented in the PR description?

> The totals in the report are 4 times the totals of a single test deck, because it is used to compare with results after scanning two precinct test decks at each precinct and two full test decks at the central scanner.

Also just curious if all counties do L&A in this way, e.g. what if a county doesn't plan on using precinct scanners or a central scanner?